tests/examples/source/devfiles/nodejs/devfile-deploy-functional-pods.yaml

```diff
@@ -113,7 +114,7 @@ components:
         spec:
           containers:
           - name: main
-            image: "{{CONTAINER_IMAGE}}"
+            image: "{{LOGS_IMAGE}}"
```

Comment:
Instead of using a specific image, it could be better to use a standard one, and to make appear what the image does in the command/args of the container:

```
          containers:
          - name: main
            image: fedora # (or lighter one)
            command: bash
            args: [ "-c", "while true; do echo \"`date` - this is infinite while loop\"; sleep 5; done" ]
```

Reply:
Makes sense. I'd prefer `alpine` vs. `fedora` because of the difference in size and also because we always pull an image.

Reply:
Addressed